docs/standard/native-interop/exceptions-interoperability.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Exceptions Interoperability"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+---
+# Working with Interop Exceptions in Unmanaged Code
```

Comment:
We do support interop for exceptions on Windows. It is well-tested area and it is fully supported, e.g. as part of managed C++.

We do not support interop for exceptions on non-Windows.

We should declare setjmp/longjmp as unsupported everywhere. It is not tested well, and it is unlikely that it works 100% even though it happens to work for simple cases.

I am sorry about the confusion ... hope that this makes it clear.

Reply:
Let me see if I got this correctly. In the first section, `Working with Interop Exceptions in Unmanaged Code`, I can add the Windows support at the beginning, making the line look something like _"Unmanaged code exception interop is not supported in non-Windows platforms, due to..."_.

Then, rephrase the second section to remove the section that states that on Windows there is support to unwind the stack and deal with external exceptions. Leave the Unix ABI not supporting section as is, as well as the unpredictable behaviors and potential crashes section.

Something like this?

Reply:
> Leave the Unix ABI not supporting section as is

Yes, but it should be moved into the "Exceptions chapter".

The section about setjmp/longjump should just say that it is not supported. It does need to explain in detail why it is not supported.